src/main/java/com/iota/iri/conf/BaseIotaConfig.java

```diff
@@ -667,6 +670,17 @@ public double getAlpha() {
     protected void setAlpha(double alpha) {
         this.alpha = alpha;
     }
+    
+    @Override
+    public boolean isTipSolidifierEnabled() {
+        return tipSolidifierEnabled;
+    }
+
+    @JsonProperty
+    @Parameter(names = "--tip-solidifier-enabled", description = SolidificationConfig.Descriptions.TIP_SOLIDIFIER_ENABLED)
```

Comment:
add `arity` please

look at other boolean properties and #930

Reply:
Should be just `--tip-solidifier` and accept `true` and `false` for enabling and disabling explicitly. 

This is done as:
@Parameter(names = "--tip-solidifier-enabled", description = SolidificationConfig.Descriptions.TIP_SOLIDIFIER_ENABLED, arity = 1)